File: Java/src/SumRange.java
import java.util.Arrays;

public class SumRange {
    public static void main(String[] args) {
        int[] ints = {1,20,3,10,100,-2,110};
        int result = sunRange(ints);
        System.out.println(result);

    }

    public static int sunRange(int[] ints) {

        return Arrays.stream(ints)
                .filter(n -> n >= 10 && n <= 100)
                .sum();
    }
}

File: Java/kotlin/SumRange.kt
package kotlin

object SumRange {
    @JvmStatic
    fun main(args: Array<String>) {
        val ints = intArrayOf(1, 20, 3, 10, 100, -2, 110)
        val result = sumRange(ints)
        println(result)
    }

    fun sumRange(ints: IntArray): Int {
        return ints.filter { n -> n in 10..100 }.sum()
    }
}

